tests/postgres_tests/test_json.py

```diff
@@ -259,6 +259,20 @@ def test_regex(self):
     def test_iregex(self):
         self.assertTrue(JSONModel.objects.filter(field__foo__iregex=r'^bAr$').exists())
 
+    def test_update(self):
+        JSONModel.objects.filter(field__k__l='m').update(field__k__l='priyansh')
```

Comment:
Please follow the style of existing tests and don't use your name for test values.

Reply:
OK

Reply:
OK